core/import-shearwater.c

```diff
@@ -97,8 +97,24 @@ extern int shearwater_profile_sample(void *handle, int columns, char **data, cha
 		cur_sample->ndl.seconds = atoi(data[4]) * 60;
 	if (data[5])
 		cur_sample->cns = atoi(data[5]);
-	if (data[6])
-		cur_sample->stopdepth.mm = metric ? atoi(data[6]) * 1000 : feet_to_mm(atoi(data[6]));
+	if (data[6]) {
+		if (atoi(data[6]) > 0) {
+			cur_sample->stopdepth.mm = metric ? atoi(data[6]) * 1000 : feet_to_mm(atoi(data[6]));
+			cur_sample->in_deco = 1;
+		} else if (data[7]) {
+			if (atoi(data[7]) > 0) {
+				cur_sample->stopdepth.mm = metric ? atoi(data[7]) * 1000 : feet_to_mm(atoi(data[7]));
+				if (data[8]) {
+					cur_sample->stoptime.seconds = atoi(data[8]) * 60;
+				}
```

Comment:
no need for extra `{ }` around `if`.

Reply:
With the change to avoid multiple atoi calls, this is now needed :D